src/Integration.UnitTests/Exclusions/AnalyzableFileIndicatorTests.cs

```diff
@@ -142,6 +142,28 @@ public void ShouldAnalyze_HasInclusions_NoExclusions_ReturnsIfIncluded(bool incl
             result.Should().Be(expectedResult);
         }
 
+        [TestMethod]
```

Comment:
If a test is directly related to a specific bug we often put a comment to help with traceability e.g. 

> // Regression test for https://github.com/SonarSource/sonarlint-visualstudio/issues/3075